src/components/templates/TalkShowStudio/TalkShowStudio.tsx

```diff
@@ -94,56 +94,56 @@ export const TalkShowStudio: FC<TalkShowStudioProps> = ({ venue }) => {
       ),
     [userOnStageSharingScreen?.data, remoteUsers, venue.id]
   );
-  const remoteCameraTrack = useMemo(
-    () =>
-      remoteUsers.find(
-        ({ uid }) =>
-          `${uid}` ===
-          userOnStageSharingScreen?.data?.[`${venue.id}`]?.cameraClientUid
-      ),
-    [userOnStageSharingScreen?.data, remoteUsers, venue.id]
-  );
```

Comment:
Why was `remoteCameraTrack` removed from here?

Reply:
This is a duplicate of changes made in https://github.com/sparkletown/sparkle/pull/1653/files#r659516038